boa/vm/fork.py

```diff
@@ -90,6 +90,10 @@ def fetch(self, method, params):
         (res,) = self.fetch_multi([(method, params)])
         return res
 
+    def fetch_uncached(self, method, params):
+        # due to __new__, underlying rpc is never CachingRPC
+        return self._rpc.fetch(method, params)
```

Comment:
we could make it part of the `RPC` api, and then i think here we just dispatch to `self._rpc.fetch_uncached`

Reply:
This is only applicable to the CachingRPC, the other ones do not cache anything.
Also note the CachingRPC is only used in the AccountFork and it's always guaranteed to be a CachingRPC.
Why do you think this should be part of the RPC API?

Reply:
> Why do you think this should be part of the RPC API?
it makes the implementation slightly cleaner (we don't need that comment anymore!). i also think it's pretty clear what it's for, like any RPC implementation might be free to cache results, and if you really want the uncached result you can use `fetch_uncached`.

Reply:
I've lifted the implementation to the base class, but the comment is still needed atm